script/crossbinary

```diff
@@ -1,6 +1,29 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
Yep, for better portability basically (see also [this SO answer](http://stackoverflow.com/a/21613044)). On my Mac, `/bin/bash` points to the old 3.x Mac bash version that nobody wants to use.